### ISSUE TITLE
Error calling getProvidersForModule

### ISSUE THREAD

**Issue by chris-peerman** [markabrahams/node-net-snmp#81]
I tried loading a MIBS file however got the following error when calling `getProvidersForModule`.

```
TypeError: Cannot read property 'SYNTAX' of undefined
    at ModuleStore.getProvidersForModule (/home/vmuser/projects/prototypes/xyz/node_modules/net-snmp/index.js:2948:24)
    at Object.<anonymous> (/home/vmuser/projects/prototypes/xyz/agent.js:73:19)
```

The line in question looks like:

```javascript
				while ( currentTableProvider || i >= entryArray.length ) {
					i++;
					mibEntry = entryArray[i];
```

However as the next line increments the counter `i` it over flows the array. It looks like it should be more like the following to check if mibEntry is defined or improve the bounds checking:
```javascript
				while ( currentTableProvider || i >= entryArray.length ) {
					i++;
					mibEntry = entryArray[i];
					if (!mibEntry) break;
```

**Comment by markabrahams** [comment]
Thanks for reporting this @chris-peerman !

I've fixed this in master now and published in version 2.6.1 of the npm.

**Comment by chris-peerman** [comment]
Thanks for fixing this :)
